index.html

```diff
@@ -0,0 +1,393 @@
+                    <li class="main-menu-list l-items"><a href="">News</a></li>
+                    <li class="main-menu-list l-items"><a id="home_campaign_btn" href="">Campaign</a></li>
+                </ul>
+            </nav>
+        </div>
+    </div>
+    <!-- ================End of The Main Container=============== -->
+   </header>
+    <!-- ===============Start of The Intro Section================ -->
+   <section class="container section-intro">
+    <div class="headline">
+        <div class="home-title">
+            <img src="./assets/img/home-title.png" alt="Home Page Title">
+            <!-- <p>"Hello, World!"</p>
+            <h1>CREATIVE CODERS<br>ONLINE SUMMIT<br>2022</h1> -->
+        </div>
+        <div class="body-txt">
+            <p>
+                A joyous festival in South Korea in October where 
+                people from more than 80 countries who believe in the 
+                value of sharing and openness and are creating positive 
+                change come together to share their thoughts.
+            </p>
+        </div>
+        <div class="location-txt">
+            <h2>2022.08.15(Sat) ~ 16(Sun)</h2>
+            <p>@National Museum of Korea, Art Center Nabi and more</p>
+        </div>
+    </div>
+   </section>
+    <!-- ================End of The Intro Section================= -->
+
+    <!-- ==============Start of The Programs Section=============== -->
+   <section class="container section-programs">
+    <div id="programs-title" class="section-title">
+        <h2>Major Programs</h2>
+        <div class="underline-bar"></div>
+    </div>
+    <div class="container program-cards">
+        <div id="program1" class="card">
+            <div class="card-content">
+                <h2 class="card-title">Something Awesome</h2>
+                <p class="card-details">Lorem ipsum
+                    dolor sit amet consectetur
+                    adipisicing elit. Fugiat rem
+                    facilis.
+                </p>
+                <a href="" class="card-btn">Learn More</a>
+            </div>
+        </div>
+        <div id="program2" class="card">
+            <div class="card-content">
+                <h2 class="card-title">Something Awesome</h2>
+                <p class="card-details">Lorem ipsum
+                    dolor sit amet consectetur
+                    adipisicing elit. Fugiat rem
+                    facilis.
+                </p>
+                <a href="" class="card-btn">Learn More</a>
+            </div>
+        </div>
+        <div id="program3" class="card">
+            <div class="card-content">
+                <h2 class="card-title">Something Awesome</h2>
+                <p class="card-details">Lorem ipsum
+                    dolor sit amet consectetur
+                    adipisicing elit. Fugiat rem
+                    facilis.
+                </p>
+                <a href="" class="card-btn">Learn More</a>
+            </div>
+        </div>
```

Comment:
- [ ] Hover-text animation looks really cool, but you should follow the design template for this section too. Please make the text visible as seen in the design template.

Reply:
Thanks Eng. @sinansevgi  for the great feedback. In regards to the hover functionality I did implemented a **@media(hover)** in case you are browsing the website with touch devices, in this case the hover ability will stop automatically and the text will be shown. Please try to open the website in mobile-mode and see yourself the text will be shown automatically. And by the way, the main purpose of this project is to show the ability of applying basic HTML , CSS and JAVASCRIPT and I did show above that.

Reply:
Hi @ZikaZaki, I noticed that, nice touch 👍 But as you can from the design template, the text should be visible on the desktop version too.
![Screenshot 2022-08-08 at 18-24-32 CC Global Summit 2015](https://user-images.githubusercontent.com/1642014/183454417-671a5a81-3d23-488c-9677-0c421e6b811e.png)

Reply:
Hi Eng. @sinansevgi. I needed to tell you that I  added the video link to the README.md file. Finally, I wanna thank you again for the great feedback.